ziti/edge/impl/conn.go

```diff
@@ -205,7 +205,9 @@ func (conn *edgeConn) HandleClose(channel2.Channel) {
 }
 
 func (conn *edgeConn) Connect(session *edge.Session, options *edge.DialOptions) (edge.Conn, error) {
-	logger := pfxlog.Logger().WithField("connId", conn.Id())
+	logger := pfxlog.Logger().WithField("connId", conn.Id()).
+		WithField("sessionId", session.Id).
+		WithField("sessionToken", session.Token)
```

Comment:
should we put tokens in the log?

Reply:
I'll remove it. Some places we only have the token, but since we have both here, we can log just the id and cross reference if necessary